qutebrowser/completion/models/base.py

```diff
@@ -39,8 +39,14 @@ class BaseCompletionModel(QStandardItemModel):
 
     Used for showing completions later in the CompletionView. Supports setting
     marks and adding new categories/items easily.
+
+    Class Attributes:
+        COLUMN_WIDTHS: The width percentages of the columns used in the
+                        completion view.
```

Comment:
I think this should be set in module-scope, not as class attribute - after all, it's  not really specific to the class.